src/account-link.ts

```diff
@@ -10,3 +11,8 @@ export function createAccountLinkForChain(address: string, chainId: string): str
   const prefix = prefixForChain(chainId)
   return prefix !== null ? `https://${prefix}etherscan.io/address/${address}` : '';
 }
+
+export function createCustomAccountLink(address: string, customNetworkUrl: string): string {
+    let parsedUrl = addPathToUrl(customNetworkUrl, 'address', address)
+    return parsedUrl;
+}
```

Comment:
Nit: The indentation is off here (we really need a linter in this repo :sweat_smile: ). Also the trailing newline was removed. I've swapped out `let` for `const` as well, since the value is never re-assigned.

```suggestion
  const parsedUrl = addPathToUrl(customNetworkUrl, 'address', address)
  return parsedUrl
}

```